acceptance/tests/invalid_ssl_config.rb

```diff
@@ -29,9 +29,9 @@
   end
 
   step 'Create alternate CA on the master and copy to agent temp folder' do
-    master_temp_ca_dir = master.tmpdir('alternate_ssl')
+    master_temp_ca_dir = master.tmpdir('alternate_ca')
     master_temp_conf = File.join(master_temp_ca_dir, 'puppet.conf')
-    create_remote_file(master, master_temp_conf, "ssldir = #{master_temp_ca_dir}")
+    create_remote_file(master, master_temp_conf, "cadir = #{master_temp_ca_dir}\nssldir = #{master_temp_ca_dir}")
```

Comment:
Ah, so configuring both `cadir and ssldir` to the same location yields the "old" behavior?

Reply:
Yes, before puppetserver 7 the cadir and ssldir were in the same location. This change preserves that behavior, as the new behavior (separating them out into `etc/puppetlabs/puppet/ssl/ca` and
`/etc/puppetlabs/puppetserver/ca`) doesn't really matter for the sake of this test.